sos/collector/clusters/ocp.py

```diff
@@ -120,6 +120,11 @@ def cleanup(self):
         """
         if self.project:
             ret = self.exec_primary_cmd("oc delete project %s" % self.project)
+            if not ret['status'] == 0:
+                self.log_error("Error deleting temporary project: %s"
+                               % ret['output'])
+            ret = self.exec_primary_cmd("oc wait namespace/%s --for=delete "
+                                        "--timeout=30s" % self.project)
             if not ret['status'] == 0:
                 self.log_error("Error deleting temporary project: %s"
                                % ret['output'])
```

Comment:
Let's change the error logged follow an `oc wait` timeout to be a bit more explicit, since an error on the delete command with the same message is handled directly before this.

e.g. "Timeout exceeded while waiting for temporary project to be deleted" or somesuch.